### PR TITLE
Remove starting generators from voltage control

### PR DESCRIPTION
**Pull request by annetill**
Signed-off-by: Anne Tilloy <anne.tilloy@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfGenerator.java [hunk 1]
```diff
@@ -211,6 +211,11 @@ protected boolean checkVoltageControlConsistency(boolean reactiveLimits, LfNetwo
             report.generatorsDiscardedFromVoltageControlBecauseNotStarted++;
             consistency = false;
         }
+        if (getTargetP() < getMinP() && getMinP() > POWER_EPSILON_SI) {
```
Comment (geofjamg) [inline src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfGenerator.java:214]: What is the meaning of "getMinP() > POWER_EPSILON_SI" ?

Comment (annetill) [reply src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfGenerator.java:214]: It means that this condition means something only for positive Pmin. For negative Pmin, I don’t know what is a starting generator…

Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfGenerator.java:214]: yes, but my question wa more about the POWER_EPSILON_SI: "getMinP() > 0"  or even maybe "getMinP() >= 0" would make sense too.

Comment (annetill) [reply src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfGenerator.java:214]: Indeed I just have copied the lines above in the code, that contains that... the good assertion is maybe Pmin > 0.

**Review by geofjamg — APPROVED** [review]

**Pull request merged**